src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java

```diff
@@ -33,9 +36,11 @@
  * Please check the contents of the license located in
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
-public class ItemPipeColoringTool extends Item {
+public class ItemPipeColoringTool extends Item implements IEnergyContainerItem {
```

Comment:
Cant you just extend ItemEnergyContainer or itemBase? Or create a new ItemPowered class?